android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1381,6 +1409,34 @@ public void initBraveNewsController() {
         }
     }
 
+    @Override
+    public void performOnConfigurationChanged(Configuration newConfig) {
+        super.performOnConfigurationChanged(newConfig);
+
+        Tab currentTab = getActivityTab();
+        if (currentTab != null
+                && currentTab.getUrl() != null
+                && isYTVideoUrl(currentTab.getUrl())
+                && !isInPictureInPictureMode()) {
+            BackgroundVideoPlaybackTabHelper.sendOrientationChangeEvent(
+                    currentTab.getWebContents(),
+                    newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE);
+        }
+    }
+
+    private boolean isYTVideoUrl(GURL url) {
```

Comment:
@tapanmodh can we place this function into some util class ? i think we are already using similar function at other places

Reply:
we are using this function in two places in `BraveActivity` class. we can place into util class but it is not being used by any another class. I think we should move after it being used by another class.

Reply:
we need some kind of test for this so would it be easier to unit test if we put it someplace else?